authorize/check_response.go

```diff
@@ -298,7 +299,7 @@ func (a *Authorize) requireWebAuthnResponse(
 	// If we're already on a webauthn route, return OK.
 	// https://github.com/pomerium/pomerium-console/issues/3210
 	if checkRequestURL.Path == urlutil.WebAuthnURLPath || checkRequestURL.Path == urlutil.DeviceEnrolledPath {
-		return a.okResponse(result.Headers), nil
+		return a.okResponse(result.Headers, nil), nil
```

Comment:
Should we wire up the `HeadersToRemove` here too, just for consistency?

```suggestion
		return a.okResponse(result.Headers, result.HeadersToRemove), nil
```

Reply:
sure